warp/cli/warp_cli.py

```diff
@@ -64,10 +77,11 @@ def invoke(program, address, function, inputs):
 )
 @click.option("--constructor_args", required=False, default="[]")
 def deploy(program, constructor_args):
-    """Deploy PROGRAM.
-
-    PROGRAM is the path to the transpiled program JSON file.
+    """
+    PROGRAM: path to the transpiled program JSON file.
 
+    Compiles the given Cairo program info file and deploys it to StarkNet.
+    Contract's address will be printed to stdout and saved in a file called CONTRACT_NAME_ADDRESS.txt
```

Comment:
I think it's no longer the case that address is saved to a file. It's just printed to stdout